src/Glpi/Application/View/Extension/RoutingExtension.php

```diff
@@ -89,16 +109,30 @@ public function path(string $resource): string
      *
      * @return string
      */
-    public function url(string $resource): string
+    public function url(string $resource, array $parameters = []): string
     {
+        if ($this->router) {
+            try {
+                // Symfony's router must take precedence over GLPI's router, for forwards compatibility
+                return $this->router->generate($resource, $parameters, UrlGeneratorInterface::ABSOLUTE_URL);
+            } catch (\Throwable) {
```

Comment:
Won't that hide potential issues?

Reply:
We want to have a unique method to generate both:
 - URL from a route name + parameters
 - URL from a script path + parameters

This method itself cannot know if the `$resource` param is a route name or a script path, so we have to try to generate the URL to figure it out.

But we should probably catch only the `RouteNotFoundException` and let the `MissingMandatoryParametersException`, `InvalidParameterException` or any other exception be thrown to not hide potential issues.

Reply:
👍

Reply:
I changed Throwable for RouteNotFoundException 👌